src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -53,15 +56,28 @@ public ApiMethodTransformer() {
   }
 
   public StaticLangApiMethodView generatePagedFlattenedMethod(
-      MethodTransformerContext context, ImmutableList<Field> fields) {
+      MethodTransformerContext context,
+      ImmutableList<Field> fields,
+      ApiMethodType type,
+      List<RequestObjectParamView> additionalParams,
+      List<ParamDocView> additionalParamDocs,
+      List<String> returnsDocLines,
+      boolean isAsync,
+      Map<String, PageStreamingDescriptorClassView> pagedByName) {
```

Comment:
I see what you're saying about how this method turned up. I will get back to you on a suggestion with a better approach.